src/linux/init/init.cpp

```diff
@@ -236,6 +242,61 @@ int WslEntryPoint(int Argc, char* Argv[])
     return ExitCode;
 }
 
+int GenerateUserSystemdUnits(int Argc, char** Argv)
+{
+    if (Argc < 2)
+    {
+        LOG_ERROR("Unit folder missing");
+        return 1;
+    }
+
+    const auto* installPath = Argv[1];
+
+    try
+    {
+        std::string automountRoot = "/mnt";
+        wil::unique_file File{fopen("/etc/wsl.conf", "r")};
+        if (File)
+        {
+            std::vector<ConfigKey> ConfigKeys = {
+                ConfigKey(wsl::linux::c_ConfigAutoMountRoot, automountRoot),
+
+            };
+            ParseConfigFile(ConfigKeys, File.get(), CFG_SKIP_UNKNOWN_VALUES, STRING_TO_WSTRING(CONFIG_FILE));
+            File.reset();
+        }
```

Comment:
Could this be passed as another argument to avoid having to parse the config file again?

Reply:
Same question for below helper.

Reply:
Unfortunately not. This code is hit when the systemd generator runs, so we can't pass arguments to it